README.md

```diff
@@ -313,9 +341,16 @@ Cakan, C., Jajcay, N. & Obermayer, K. neurolib: A Simulation Framework for Whole
 
 Caglar Cakan (cakan@ni.tu-berlin.de)  
 Department of Software Engineering and Theoretical Computer Science, Technische Universität Berlin, Germany  
-Bernstein Center for Computational Neuroscience Berlin, Germany  
+Bernstein Center for Computational Neuroscience Berlin, Germany 
+
+For questions or comments on the optimal control module, please contact
+Lena Salfenmoser (lena.salfenmoser@tu-berlin.de)
+Department of Software Engineering and Theoretical Computer Science, Technische Universität Berlin, Germany  
+
 
 ### Acknowledgments
 This work was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) with the project number 327654276 (SFB 1315) and the Research Training Group GRK1589/2.
 
+The optimal control module was developed and implemented by Lena Salfenmoser and Martin Krück, and supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) with the project number 163436311 (SFB 910).
+
```

Comment:
```suggestion
The optimal control module was developed by Lena Salfenmoser and Martin Krück supported by the DFG project 163436311 (SFB 910).
```

This is a better place for Acknowledgment.

Reply:
I was basically copy-pasting what you wrote, just adapting names and project numbers... but I'm also fine with your suggestion